pkg/props/color.go

```diff
@@ -1,14 +1,18 @@
-// Package contains all props used to customize components.
 package props
 
 import "fmt"
 
 var (
+	// WhiteColor is a Color with all values in 255.
 	WhiteColor = Color{Red: 255, Green: 255, Blue: 255}
+	// BlackColor is a Color with all values in 0.
 	BlackColor = Color{Red: 0, Green: 0, Blue: 0}
-	RedColor   = Color{Red: 255, Green: 0, Blue: 0}
+	// RedColor is a Color with all values in 255.
```

Comment:
// RedColor is a Color with only Red in 255.

Reply:
done